Detectors/ITSMFT/ITS/tracking/src/TimeFrame.cxx

```diff
@@ -609,16 +609,19 @@ void TimeFrame::printNClsPerROF()
   }
 }
 
-void TimeFrame::printROFInfo(const int rofId)
+void TimeFrame::printSliceInfo(const int startROF, const int sliceSize)
 {
-  std::cout << "ROF " << rofId << " dump:" << std::endl;
-  for (int iLayer{0}; iLayer < mClusters.size(); ++iLayer) {
-    std::cout << "Layer " << iLayer << " has: " << getClustersOnLayer(rofId, iLayer).size() << " clusters." << std::endl;
-  }
-  std::cout << "Number of seeding vertices: " << getPrimaryVertices(rofId).size() << std::endl;
-  int iVertex{0};
-  for (auto& v : getPrimaryVertices(rofId)) {
-    std::cout << "\t vertex " << iVertex++ << ": x=" << v.getX() << " " << " y=" << v.getY() << " z=" << v.getZ() << " has " << v.getNContributors() << " contributors." << std::endl;
+  std::cout << "Dumping slice of " << sliceSize << " rofs:" << std::endl;
+  for (int iROF{startROF}; iROF < startROF + sliceSize; ++iROF) {
+    std::cout << "ROF " << iROF << " dump:" << std::endl;
+    for (int iLayer{0}; iLayer < mClusters.size(); ++iLayer) {
```

Comment:
this will pop up at next CI as signed/unsigned comparison, could you cast the `size` to `int`?